cinema/views.py

```diff
@@ -1,138 +1,39 @@
-from datetime import datetime
-
-from django.db.models import F, Count
 from rest_framework import viewsets
-from rest_framework.pagination import PageNumberPagination
-
+from cinema.serializers import MovieSerializer
```

Comment:
The import statement for `MovieSerializer` is correct, but ensure that all necessary serializers are imported if other viewsets require them.